README.md

```diff
@@ -1,5 +1,7 @@
 ![logo](https://static.creatordev.io/logo-md-s.svg)
 
+[![Build Status](http://jenkins.creatordev.io/buildStatus/icon?job=CreatorDev/w1-switchcounter/master)](http://jenkins.creatordev.io/job/CreatorDev/w1-switchcounter/master)
```

Comment:
this link won't navigate to the job when you click on the button (the 2nd URL) - it just 404s. It would need to be changed to http://jenkins.creatordev.io/job/CreatorDev/job/w1-switchcounter/job/master/ I think. Our main openwrt repo also seems to be incorrect (it 404s when you click it).